.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -26,10 +27,18 @@ assignees: ''
 
 <!-- Add any other context or screenshots about the feature request here. -->
 
+---
+
+#### If you enjoy Lightning, check out our other projects! ⚡
+
+<sub>
+
+- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.
+
+- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
+
```

Comment:
```suggestion
- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
```
list is without spaces, right?

Reply:
The `sub` block doesnt render properly without the spaces